assets/src/edit-story/components/checklist/index.js

```diff
@@ -58,6 +65,15 @@ export function Checklist() {
     })
   );
 
+  const [openPanel, setOpenPanel] = useState(null);
+  const handleOpenPanel = useCallback(
+    (panelName) => () =>
+      setOpenPanel((currentOpenPanel) =>
+        currentOpenPanel === panelName ? null : panelName
+      ),
+    []
+  );
```

Comment:
Should we open a panel initially? Right now it just opens to this:
<img src="https://user-images.githubusercontent.com/22185279/123706948-32594480-d826-11eb-8715-4a2917896087.png" height="300px" />

Reply:
Yes! so this is going to need to mirror the logic currently in the old checklist. 

So no panels at story start
Then when the 'recommened' checkpoint is hit (that's now design and accessibility) those are available and design is open. 

then when priority checkpoint is hit the priority tab opens by default.

Reply:
So I thought this was included in another ticket but alas, I cannot find it anymore. Made a new one for that here: https://github.com/google/web-stories-wp/issues/8114

Reply:
ah okay, thanks! I'll wait to do that then